vyper/ast/grammar.lark

```diff
@@ -26,7 +26,8 @@ WILDCARD: "*"
 _import_name: NAME
 // NOTE: Don't use DOT here, just a separator
 _import_path: (_import_name ".")* _import_name
-import_alias: _AS NAME
+_import_alias_name: NAME
```

Comment:
why is this necessary?

Reply:
I have included the explanation in my reply to the next comment.